dcat/index.html

```diff
@@ -2495,9 +2495,12 @@ <h4>Property: in series</h4>
                 <!--tr><th class="prop">Inverse property of:</th>
                 <td><a href="#Property:dataset_series_series_member"><code>dcat:seriesMember</code></a></td>
                 </tr-->
-                <tr><th class="prop">Usage note:</th><td> Normally, child datasets in dataset series are represented as <code>dcat:Dataset</code>. The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.</td></tr>
+                <!--<tr><th class="prop">Usage note:</th><td> Normally, child datasets in dataset series are represented as <code>dcat:Dataset</code>.The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.></td></tr-->
                 <!--tr><th class="prop">See also:</th><td>Sub-properties of <code>dcterms:hasPart</code> in particular <a href="#Property:catalog_dataset"><code>dcat:dataset</code></a>, <a href="#Property:catalog_catalog"><code>dcat:catalog</code></a>, <a href="#Property:catalog_service"><code>dcat:service</code></a>. </td></tr-->
                 </table>
+
+<p>For guidance on the use of this property, see <a href="#dataset-series-before-dcat3"></a>.</p>
```

Comment:
@andrea-perego : I wonder if we should point to the section #dataset-series instead of the subsection about  existing implementations.

```suggestion
<p>For guidance on the use of this property, see <a href="#dataset-series"></a>.</p>
```

Reply:
Agreed.